deploy/kubernetes/role.yaml

```diff
@@ -39,7 +39,7 @@ rules:
   resources: [deployments, endpoints, ingressclasses, nodes, pods/log, replicationcontrollers, secrets]
   verbs: [get, list, watch]
 - apiGroups: [vmware.com]
-  resources: [loadbalancers, loadbalancers/status, nsxlbmonitors, nsxlbmonitors/status]
+  resources: [loadbalancers, loadbalancers/status, nsxlbmonitors, nsxlbmonitors/status, virtualnetworkinterfaces, virtualnetworkinterfaces/status, virtualnetworks, virtualnetworks/status]
```

Comment:
They are for WCP CRDs, do you think we should add them?

Reply:
In NCP yaml the logic is:
```
    142    {% if nsxapi == "policy" %}
    143      - virtualnetworks
    144      - virtualnetworks/status
    145      - virtualnetworkinterfaces
    146      - virtualnetworkinterfaces/status
    147    {% endif %}
```
Looks that it's not only for WCP.